org.arrl.trustedqsl.json

```diff
@@ -0,0 +1,148 @@
+{
+    "app-id" : "org.arrl.trustedqsl",
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "22.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "command" : "tqsl",
+    "finish-args" : [
+	"--share=network",
+	"--socket=x11",
+	"--filesystem=home"
+     ],
+    "modules" : [
+        {
+            "name" : "openssl",
```

Comment:
I will repeat what I said on the PR you closed: you do NOT need to build either openssl or curl.

Reply:
Well, duh. You're absolutely correct - a build without those works just fine. 
I'm removing this.